tests/Functional/ContentProjectRenderTest.php

```diff
@@ -157,35 +149,47 @@ public function testProjectRendering(): void {
     $this->assertFeaturedMediaField($project_details, 'project_featured_media');
 
     // Assert the description blocks inside the Project details.
-    $description_lists = $project_details->findAll('css', 'dl.ecl-description-list.ecl-description-list--horizontal.ecl-description-list--featured');
+    $description_lists = $project_details->findAll('css', '.ecl-col-12.ecl-col-md-6.ecl-u-mt-l.ecl-u-mt-md-none .ecl-u-mb-s');
     $this->assertCount(3, $description_lists);
 
     // Assert the first description list block's labels and values.
-    $labels = $description_lists[0]->findAll('css', 'dt.ecl-description-list__term');
-    $this->assertCount(3, $labels);
-    $this->assertEquals('Reference', $labels[0]->getText());
-    $this->assertEquals('Project duration', $labels[1]->getText());
-    $this->assertEquals('Project locations', $labels[2]->getText());
-    $values = $description_lists[0]->findAll('css', 'dd.ecl-description-list__definition');
-    $this->assertCount(3, $values);
-    $this->assertEquals('Project reference', $values[0]->getText());
-    $this->assertEquals('10.05.2020 - 15.05.2025', $values[1]->getText());
-    $this->assertContains('09199 Ages Burgos, Spain', $values[2]->getText());
-    $this->assertContains('Munich, Germany', $values[2]->getText());
+    $field_list_assert = new FieldListAssert();
+    $first_field_list_expected_values = [
+      'items' => [
+        [
+          'label' => 'Reference',
+          'body' => 'Project reference',
+        ], [
+          'label' => 'Project duration',
+          'body' => "10.05.2020\n - 15.05.2025",
+        ], [
+          'label' => 'Project locations',
+          'body' => "09199 Ages Burgos, Spain\n\n  Munich, Germany",
+        ],
+      ],
+    ];
+    if (version_compare(PHP_VERSION, '7.3') < 0) {
```

Comment:
As I understand that change based on that decision: https://github.com/openeuropa/oe_theme/pull/629#discussion_r531058337
Should we add here a comment for describing the problem and a link to a follow-up ticket for better tracking?

Reply:
Answered above.